packages/frontend/src/components/common/Filters/FilterInputs/DefaultFilterInputs.tsx

```diff
@@ -35,6 +36,13 @@ const DefaultFilterInputs = <T extends ConditionalRule>({
             ? rule.disabled && !rule.values
             : undefined,
     });
+    const inputRef = useRef<HTMLInputElement>(null);
+
+    useEffect(() => {
+        if (inputRef.current) {
+            inputRef.current.focus();
```

Comment:
Have you tried using the `autofocus` HTML property instead of applying focus programatically?